### ISSUE TITLE
Scss is not reloaded until the server is restarted

### ISSUE THREAD

**Issue by barcher** [DudaDev/ember-cli-sass-pods#17]
version: 2.4.2
node: 5.7.1
os: darwin x64

```
app/styles/
├── app.scss
└── pods.scss
```

Importing pods into app and `...test/style` into pods.

```
app/pods/components/
├── test
    ├── component.js
    ├── style.scss
    └── template.hbs
```

```
//ember-cli-build.js
module.exports = function(defaults) {
  var app = new EmberApp(defaults, {
    sassOptions: {
      includedPaths: ['app']
    }
  });
```

I've also tried `includedPaths: ['app/pods']` and globbing. Thanks!

**Comment by justtal** [comment]
How does your pods.scss looks like?

Does it have import of the pods/components/test/style.scss?

It supposed to be automatically if you used 'ember g style' command.

**Comment by barcher** [comment]
Yep, it imports the `pods/components/test/style.scss` styles. I've tested this by restarting the server / modifying the styles in that file. I feel like there may be something weird going on with livereload since writing to the `pods.scss` file will also forcefully reload the styles, but is not detecting changes in the individual pods.

**Comment by justtal** [comment]
I didn't understand.

If you're set this style:

`body {background: red}`

to your pods.scss file. It will reload automatically? with the actual change?

**Comment by barcher** [comment]
Yes, but only if the `pods.scss` file is manually written. Updating a child stylesheet does not rewrite or update `pods.scss` which manifests itself as no visible changes until 1) the server is reloaded, or 2) `pods.scss` is manually written.

**Comment by MartinMalinda** [comment]
Experiencing this as well. Must be an issue with some new Ember-CLI. If a scss file outside of /styles folder is changed, dist css folder is not updated even if the "build" is triggered.

**Comment by MartinMalinda** [comment]
@barcher I have managed to solve my issue. Not sure how exactly though. 
See here:
https://github.com/ember-cli/ember-cli/issues/5823

**Comment by barcher** [comment]
@MartinMalinda thanks! I tried adding the `sassOptions` as you suggest, but it doesn't seem to affect the results.
